### DISCUSSION TITLE
IslandChatEvent and IslandBanEvent

### DISCUSSION THREAD

**Discussion by Casheh** [BG-Software-LLC/SuperiorSkyblock2#239] [Implemented Feature Requests] [answered]
The IslandRestricMoveEvent works as a ban event, however it will only be called if the target is moving at the time, which isn't always practical. As for the chat event, it would be great if devs had a way to set the cancellations of certain messages said in island chat.

**Comment by OmerBenGera** [comment, marked as answer]
After a lot of time, finally it was added.
